core/lib/spree/testing_support/factories/inventory_unit_factory.rb

```diff
@@ -5,8 +5,18 @@
 
 FactoryGirl.define do
   factory :inventory_unit, class: 'Spree::InventoryUnit' do
+    transient do
+      order nil
```

Comment:
I would consider this wrong as this implies (from reading the specs) that we could set an order on a inventory unit, what we recently prohibited.

The shipment already has the order. Could we change the specs that we set the correct order on the shipments instead?